app/models/workers_scaling/experiment_statistics_factory.rb

```diff
@@ -0,0 +1,19 @@
+module WorkersScaling
+  class ExperimentStatisticsFactory
+    @cache = ActiveSupport::Cache::MemoryStore.new expires_in: 10.minutes
```

Comment:
Ustalaliśmy, że bardziej czytelnie będzie robić nawiasy. Tutaj szczególnie, bo zarówno nawiasy, jak stworzenie hasha jest ukryte.
Podanie argumentu przy stworzeniu cache ustala to jako domyślny argument przy wszystkich zapytaniach? Nie znalazłem jasnej odpowiedzi na ten temat.

Poza tym lepiej użyć MemCacheStore, jest wydajniejsze podobno.